addon/components/rdf-input-fields/currency-input.js

```diff
@@ -0,0 +1,17 @@
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import SimpleInputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/simple-value-input-field';
+
+export default class RdfInputFieldsCurrencyInputComponent extends SimpleInputFieldComponent {
+  inputId = 'input-' + guidFor(this);
+
+  currencySymbol = '€';
+
+  @action
+  updateValue(e) {
+    if (e && typeof e.preventDefault === 'function') e.preventDefault();
+    this.value = e.target.value.trim();
+
+    super.updateValue(this.value);
```

Comment:
Question: 1 thing I'm not sure about is the data type. At the moment this will always be a string, but I'm not sure if that's the best type for currencies in RDF, but I'm no expert on that. Ignore this if string is the way to go.

Reply:
@aatauil  

Good point. Had some though about it to. But not sure how they want the value in the database. They will always build up the currency symbol with the value I assume? Didn't get an example on how this reflects to the old use in the forms.

Reply:
We could add some more values for it but maybe that's overkill? the data structure would look like this
```ttl
# Currency Input
<http://ember-submission-form-fields/source-node>
  <http://mu.semte.ch/vocabularies/ext/currencyInputValue> "0,0" ;
  <http://schema.org/currency> "€" ;
  <http://schema.org/price> "€0,0" ;
  <http://schema.org/amount> 0.0 .

Reply:
Sorry, the conclusion is a bit vague for me. String is the way to go? The chat seemed to imply that numbers would be preferred for sorting / filtering reasons?

Reply:
@JonasVanHoof Conclusion here is to just store the value as a decimal in the database ? from what I understand in rocketchat